packages/integrations/markdoc/src/index.ts

```diff
@@ -0,0 +1,107 @@
+import type { AstroIntegration, AstroConfig } from 'astro';
+import { InlineConfig } from 'vite';
+import type { Config } from '@markdoc/markdoc';
+import Markdoc from '@markdoc/markdoc';
+import { prependForwardSlash, getAstroConfigPath, MarkdocError, parseFrontmatter } from './utils.js';
+import { fileURLToPath } from 'node:url';
+import fs from 'node:fs';
+
+export default function markdoc(markdocConfig: Config = {}): AstroIntegration {
+	return {
+		name: '@astrojs/markdoc',
+		hooks: {
+			'astro:config:setup': async ({ updateConfig, config, addContentEntryType }: any) => {
```

Comment:
Any chance to not use `any` here? I've seen that sometimes we use `any` and sometimes we don't. Ideally, we should be consistent and document when we're forced to use `any`